app/src/test/shared/components/ErrorLogTests.tsx

```diff
@@ -3,19 +3,32 @@ import { shallow } from "enzyme";
 import { expect } from "chai";
```

Comment:
All that's happened here is there is one new test at the end, but the whitespace was messed up (indented with 3 spaces instead of 4) so everything has changed.